dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/request/IRequestImpl.java

```diff
@@ -103,7 +105,7 @@ public String getAuthType()
   {
       // TODO webcontainer checks dispatcher.isSecurityenabledForapplication
       // and uses some private attributes in that case
-      String type = this.conn.getTrustedHeader(HttpHeaderKeys.HDR_$WSAT.getName());
+      String type = this.conn.getTrustedHeader(HttpHeaderKeys.HDR_$WSAT);
```

Comment:
With the other changes I expect this doesn't compile.

Reply:
You make a getTrustedHeader method in this class to be able to get things working the way you want similar to what you did in HttpDispatcherLink.

Reply:
Done.